case/infrasim/virtual_node/T130052_idic_ESXiTest.py

```diff
@@ -36,9 +36,11 @@ def boot_to_disk(self, node):
         print node.ssh.send_command_wait_string(str_command="echo infrasim | sudo -S rm -f /tmp/esxi6p3-1.qcow2"+chr(13), wait="~$")
 
         dst_path = node.send_file(os.environ["HOME"]+"/images/esxi6p3-1.qcow2", "/tmp/esxi6p3-1.qcow2")
-        #dst_path = "/tmp/esxi6p3-1.qcow2"
 
-        str_node_name = node.get_instance_name()
+        try:
+            str_node_name = node.get_instance_name()
+        except:
+            self.result(BLOCK, "Failed to get infrasim instance.")
```

Comment:
In puffer, a sentence to BLOCK won't necessarily return. You need explicitly return where the test should end.
In this test situation, I believe you shall return since no node is found.